node-project-template/package.json

```diff
@@ -10,16 +10,17 @@
     "test:watch": "watch 'npm test' ./lib ./test",
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
-    "lint:tests": "./lint-tests.sh",
     "start": "node ."
   },
   "dependencies": {
-    "eslint": "^3.12.1",
+  },
+  "license": "MIT",
+  "devDependencies": {
     "@usabilla/js-styleguide": "^1.0.0",
+    "eslint-plugin-jasmine": "^2.2.0",
+    "eslint": "^3.12.1",
     "jasmine": "^2.5.2",
     "jasmine-spec-reporter": "^3.1.0",
-    "lodash": "^4.12.0",
     "watch": "^1.0.1"
-  },
-  "license": "MIT"
+  }
```

Comment:
AFAICT all these dependencies are devDependencies, so moved them there, lodash wasn't used and would be installed unnecessarily so I removed it.